app/assets/config/client_id.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"852120292135-f784j6m8909uena2q8vgkifi10un298u.apps.googleusercontent.com","project_id":"boreal-physics-162914","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://accounts.google.com/o/oauth2/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"vKckKlgswyvZzK0E0UwUEu0A","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
agreed, I was required to do it by google to run their ruby program to tie my calendars into the program